README.md

```diff
@@ -346,11 +346,55 @@ If issues are arising for the emulator, you may need to clear the caches.
 
 ### Debugging in WSL2 and Android Emulator
 
-If you're developing on Windows using WSL2, follow these setup steps:
+If you're developing on Windows using WSL2, follow these setup steps to set up an emulator running on Windows that can communicate with adb running in WSL:
 
-**Install in WSL2:**
+**Configure your Android Emulator**
+
+- Install Android Studio on your Windows host (Windows)
+   - Download [Android Studio for Windows](https://developer.android.com/studio) and go through the installer, setting up an emulator / virtual device
+- Configure your wslconfig (Windows)
+   - locate your .wslconfig file
+   >HINT: it should be in `%userprofile%\.wslconfig`
+   - include in your config the following lines
+   ```
+   [wsl2]
+   networkingmode=mirrored
+   hostAddressLoopback=true
+   ```
```

Comment:
suggestion: We should include a comment about what these config options are doing.